typescript/package.json

```diff
@@ -8,5 +8,32 @@
   "devDependencies": {
     "@types/chai": "^4.2.21",
     "@types/mocha": "^9.0.0"
+  },
+  "name": "kurtosis-js-lib",
+  "version": "1.0.0",
```

Comment:
@GoodMorningA1i you'll need to make sure this is updated as part of the `release.sh` script flow